presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/PlanNodeStats.java

```diff
@@ -50,7 +52,8 @@
             long planNodeOutputPositions,
             DataSize planNodeOutputDataSize,
             Map<String, OperatorInputStats> operatorInputStats,
-            Map<String, OperatorHashCollisionsStats> operatorHashCollisionsStats)
+            Map<String, OperatorHashCollisionsStats> operatorHashCollisionsStats,
+            Optional<WindowNodeStats> windowNodeStats)
```

Comment:
In any case, WindowNodeStats is the wrong abstraction - this class is about operators. Introducing a plan node-related member variable is weird.